### PR TITLE
making client recipe works on CentOS

### PR DESCRIPTION
**Pull request by tmoreau**

### CODE REVIEW

File: attributes/client.rb [hunk 1]
```diff
@@ -23,8 +23,8 @@
 when 'ubuntu'
   default['gluster']['client']['package'] = 'glusterfs-client'
 when 'redhat', 'centos'
-  default['gluster']['client']['package'] = 'glusterfs'
+  default['gluster']['client']['package'] = ['fuse', 'fuse-libs', 'glusterfs', 'glusterfs-fuse']
```
Comment (shortdudey123) [inline attributes/client.rb]: The only package that should be needed is `glusterfs-fuse`.  This installs `glusterfs` as a dependency.
Tested on CentOS7

Further info: http://www.unixmen.com/install-glusterfs-server-client-centos-7/

**Comment by tmoreau** [comment]
Check for the `glusterfs` package, indeed I will remove that one.

Regarding the `fuse` and `fuse-libs` packages, if those are not added at least on CentOS 6, the mount command fails.

**Comment by shortdudey123** [comment]
Can you output what happens w/o fuse on centos6?

Not having an issue w/o fuse and can't find any doc that says to install `fuse` or `fuse-lib`

**Comment by tmoreau** [comment]
I tested again and I cannot reproduce... :-(
LEt me close that pull request and create new one to fix the typo with default['gluster']['client']['volumes'] definition and adding the client::mount to the client recipe.

**Pull request closed without merging**